src/qaqc/download_and_flag_fxns/generate_summary_statistics.R

```diff
@@ -32,7 +32,7 @@ generate_summary_statistics <- function(site_param_df) {
       rollsd = roll_sd(mean, n = 7, align = 'center', na.rm = F, fill = NA_real_),
       # Determine the slope of a point in relation to the point ahead and behind.
       slope_ahead = abs(front1 - mean)/15,
-      slope_behind = abs(mean - back1)/15,
+      slope_behind = (mean - back1)/15,
```

Comment:
question: is this supposed to not be an absolute value anymore?

Reply:
Correct, I no longer want this to be the absolute slope value - see my comment above.